packages/vite/src/node/ssr/ssrTransform.ts

```diff
@@ -90,17 +90,22 @@ async function ssrTransformScript(
   }
 
   let uid = 0
-  const deps = new Set<string>()
   const dynamicDeps = new Set<string>()
   const idToImportMap = new Map<string, string>()
+  const importIdMap = new Map<string, string>()
```

Comment:
```suggestion
  const depToImportIdMap = new Map<string, string>()
```

nit: I think we should rename it to `depToImportIdMap` here. It would be nice to keep the terms and distinction of `deps` and `dynamicDeps` in this code.